### ISSUE TITLE
"Customizable Options" if defined for a product in Magento are broken after every product import from Akeneo to Magento

### ISSUE THREAD

**Issue by paul-logicspot** [akeneo/magento2-connector-community#244]
### Environment and configuration
1. "akeneo/module-magento2-connector-community" / version 100.4.0
2. Magento commerce 2.3.3 (seems not to be relevant)

### Steps to reproduce
1. Create a simple product in Akeneo
2. Import the created product to Magento
3. Edit the product in Magento Admin Panel, add some "Customizable Options", confirm the customizable options are displayed on frontend
4. (optional) Edit the product in Akeneo (update any attribute)
5. Import the product again to Magento

### Expected result
1. The "Customizable Options" are displayed on frontend

### Actual result
1. The "Customizable Options" are no longer displayed on frontend but they are still visible in the Magento Admin Panel at the product edit page. (If you save a product in Admin it solves the issue)

### Source of the issue
1. The problem is in the class Akeneo\Connector\Job\Product, function createEntities(), the following lines:
```
'has_options'      => new Expr(0),
'required_options' => new Expr(0),
```
as they always reset the values of 'has_options' and 'required_options' to 0 after every product import from Akeneo to Magento even if they are set to 1 in Magento.
Currently this is here: https://github.com/akeneo/magento2-connector-community/blob/master/Job/Product.php#L1181-L1182
Imo these lines should be just removed

**Comment by Dnd-Gimix** [comment]
Hello @paul-logicspot ,

Thank you for your interest in Akeneo Connector for Magento 2.
We have released a bug fix for this in the latest 100.4.12 version of the connector.

Could you try again and tell us if it is now OK for the custom options ?

Many thanks.

Regards,

**Comment by paul-logicspot** [comment]
I confirm the issue had been resolved in the version 100.4.12. Thanks!

**Comment by Dnd-Gimix** [comment]
Thank you @paul-logicspot !
